cmd/lotus/daemon.go

```diff
@@ -35,6 +36,11 @@ const (
 	preSealedSectorsFlag = "genesis-presealed-sectors"
 )
 
+var (
+	lotusInfo  = stats.Int64("info", "General info about lotus", stats.UnitDimensionless)
```

Comment:
Let's rename it to `version`

Reply:
This is kind of funky, but the idea was that `lotusInfo` would be the metric and then I'd pass through tags to expose other data, such as version, build, etc. We could then tag other calls using similar tags, which are selectively exposed in the view.

I can change it to version, but the counter is only there for me to tag other info on to :)

Reply:
Hmm, yeah you are right. Reading output once again `lotus_info` makes sense as we can tack more info onto it.

Sorry for asking you to change it back.

Reply:
No problem at all.